lib/server/backends/redis.ts

```diff
@@ -53,20 +64,58 @@ type Options = Partial<InstantBanditOptions & RedisBackendOptions>;
 export function getRedisBackend(initOptions: Options = {}): RedisBackend & SessionsBackend {
   const options = Object.freeze(Object.assign({}, DEFAULT_REDIS_OPTIONS, initOptions));
   const redis = new ioredis(options);
+  let connected = false;
   return {
     get client() { return redis; },
+    get connected() { return connected; },
 
-    async connect(): Promise<void> {
-      switch (redis.status) {
-        case "connecting":
-        case "connect":
-        case "ready":
+    async connect(block = false): Promise<void> {
```

Comment:
What is the purpose of this block on the connection?

Reply:
Only used in testing at the moment, can remove.